### DISCUSSION TITLE
Better highlight route line

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#437] [Q&A] [answered]
I was traveling in Namibia/South Africa last month.
With a slightly adjusted route profile from Poutnikl.
Worked without a problem!
In my opinion, the visibility of the route line could be improved. Depending on the "ambient color" of the map, the route line is often not optimally visible.
If you could "highlight" the edge of the route line with a different color, that would be good!
There should be a ca.10% border with a different color on the left and right of the route line.

**Comment by devemux86** [comment, marked as answer]
@Leeb64 thanks for the suggestion.

Please check this related discussion where we explain how it could be done,
and what will be lost (line transparency) if we use double road lines:
- https://github.com/devemux86/cruiser/discussions/421
